ui/src/test/java/org/pentaho/di/ui/trans/step/BaseStepDialog_ConnectionLine_Test.java

```diff
@@ -250,6 +251,7 @@ public void showDbDialog_LoopsUntilUniqueValueIsInput() throws Exception {
       // unique value
       .thenReturn( expectedResult );
 
+    when( databaseDialog.getDatabaseMeta() ).thenReturn( createDefaultDatabase() );
```

Comment:
What is the relation of this change and the actual bug being fixed?

Reply:
Because database dialog was mocked when it called getDatabaseMeta it returned null.  Since I'm making use of this call it would bomb the test.  This was only to fix that.